doc/advanced-configuration/Listener-modules.md

```diff
@@ -88,13 +88,16 @@ Unlike `ejabberd_c2s`, it doesn't use `ejabberd_receiver` or `ejabberd_listener`
 
             `{"localhost", "/api", mongoose_api, [{handlers, [mongoose_api_metrics]}]}`
 
-  * `mongoose_api_admin` -  REST API for admin commands.
+  * `mongoose_api_admin` -  REST API for admin commands. It expects credentials in the form of {Username, Password}
```

Comment:
Please format the new information in a way similar to `mod_websockets` (bullet point, even if it's the only parameter for now).